controllers/topics.js

```diff
@@ -19,8 +19,10 @@ const {
 
 const editTopicHandler = (req, res, next) => {
 	const context = req.originalUrl.split('/')[1];
-	let lessonPromise; let action; let
-		method;
+	let lessonPromise;
+	let action;
+	let	method;
+	const referrer = req.query.returnUrl;
```

Comment:
maybe a call it redirectUrl instead?

Reply:
due to consistency reasons with the other controllers and their behavior I would leave it as "referrer". To follow the logic with the other stuff in the legacy client I thing it would only raise the confusion if we name it differently in comparison to the other legacy logic.